doc/contributing/coding-conventions.xml

```diff
@@ -229,7 +229,12 @@ args.stdenv.mkDerivation (args // {
     </listitem>
     <listitem>
      <para>
-      If a package is not a release but a commit from a repository, then the version part of the name <emphasis>must</emphasis> be the date of that (fetched) commit. The date <emphasis>must</emphasis> be in <literal>"YYYY-MM-DD"</literal> format. Also append <literal>"unstable"</literal> to the name - e.g., <literal>"pkgname-unstable-2014-09-23"</literal>.
+      If a package is not a release but a commit from a repository, then the version part of the name <emphasis>must</emphasis> be the date of that (fetched) commit in <literal>"YYYY-MM-DD"</literal> format, followed by <literal>"-unstable"</literal>. For example, <literal>"pkgname-2014-09-23-unstable"</literal>.
```

Comment:
While we're at it, could we specify that we want the commit the date and not the authoring date?

Reply:
> I liked it better at the beginning

Here's one problem with having it at the beginning of the version:

```
builtins.parseDrvName "pname-unstable-2.0"
{ name = "pname-unstable"; version = "2.0"; }
```

Reply:
Is this commit vs authoring date distinct really necessary to have here? It seems like a detail that isn't often going to matter and it will prompt someone to have to look up the two date types and how to fetch the proper one.

Reply:
99% of the times it does not matter, but the dates _can_ be wildly different. Maybe to avoid confusion we could just say that the commit date must be used, without specifying that the authoring date must not?

Reply:
I resolved this by adding the word "commit" one more time.